snapshot/Makefile

```diff
@@ -12,13 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.PHONY: all controller
+ifeq ($(REGISTRY),)
+	REGISTRY = quay.io/kubernetes_incubator/
+endif
+ifeq ($(VERSION),)
+	VERSION = latest
+endif
+IMAGE = $(REGISTRY)snapshot-controller:$(VERSION)
+MUTABLE_IMAGE = $(REGISTRY)snapshot-controller:latest
+
+.PHONY: all controller clean container container-quick
 
 all: controller
 
-controller: 
+controller:
 	go build -o _output/bin/snapshot-controller cmd/snapshot-controller/snapshot-controller.go
 	go build -o _output/bin/snapshot-provisioner cmd/snapshot-pv-provisioner/snapshot-pv-provisioner.go
 
 clean:
+	-rm -f deploy/snapshot-controller
+	-rm -f deploy/snapshot-provisioner
 	-rm -rf _output
+
+container: controller quick-container
+
+container-quick:
```

Comment:
it's `quick-container` on the previous line

Reply:
Oops. I could not decide how to name the target... And this is the outcome. Will fix it.